workloads/logging/deploy_logging_stack.sh

```diff
@@ -28,8 +28,13 @@ fi
 
 function install() {
   # create cluster logging and elasticsearch resources
-  log "Creating cluster logging and elastisearch resources"
-  envsubst < ./files/logging-stack.yml | oc create -f -
+  if [[ $DEPLOY_ES == "true" ]]; then
```

Comment:
This will skip the logging operator/fluentd setup as well, maybe modify the logging-stack template to selectively install Elasticsearch when DEPLOY_ES is set? Thoughts?

Reply:
As it stands, if you don't set DEPLOY_ES to true or provide a custom es url then the script will pretty much just run the cleanup and then timeout waiting for the logger to come up. We could 
a) key off the custom url only and otherwise default to deploying; 
b) Allow it to setup a stub cluster logging instance with pretty much no configuration
c) Make the cleanup optional so it could verify that the cluster logging is "up"

I think option c would provide the most flexibility long term but am open to other options as well

Reply:
The intention of this variable is to have the ability to just setup the logging operator, fluentd stack when ES is not used as the backend and forward the logs to the custom ES right? This conditional will skip the entire logging stack install like you mentioned. We can tweak the spec to conditionally skip the elasticsearch setup when custom ES url is defined: https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/logging/files/logging-stack.yml#L82. Thoughts?

Reply:
@chaitanyaenr I inverted the if on the install to simply be if the custom es url is defined use it otherwise do the install per normal. Does that resolve your concerns?